ansible/group_vars/all.yml

```
@@ -0,0 +1,57 @@
+migration:
+  default_transfer_method: "rsync"  # rsync, zfs
+  verify_data_integrity: true
+  create_backup_before_migration: true
+  migration_timeout_minutes: 60
+
+# Backup settings
+backup:
+  default_retention_days: 7
+  backup_compression: true
+  verify_backups: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)